WcaOnRails/app/models/user.rb

```diff
@@ -604,6 +614,10 @@ def can_view_delegate_matters?
     any_kind_of_delegate? || can_admin_results? || wrc_team? || wdc_team? || quality_assurance_committee? || competition_announcement_team?
   end
 
+  def can_view_wdc_page?
+    admin? || any_kind_of_staff?
```

Comment:
This should be `admin? || staff?`, no?

Reply:
Ah, good eye! This method actually isn't used anywhere, and shouldn't! Anyone can view the wdc page. I must have written this when I started working on this project, stopped using it at some point, but never thought to actually remove it. Whooops.